docs/security/08-02-update-tls-certificates.md

```diff
@@ -22,16 +22,10 @@ The TLS certificate is a vital security element. Follow this tutorial to update
 
   >**CAUTION:** When you regenerate the TLS certificate for Kyma, the `kubeconfig` file generated through the Console UI becomes invalid. To complete these steps, use the admin `kubeconfig` file generated for the Kubernetes cluster that hosts the Kyma instance you're working on.
 
-  1. Edit the `owndomain-overrides` ConfigMap and replace the existing certificate and key found in the **global.tlsCrt** and **global.tlsKey** with a new, base64-encoded pair.
+  1. Delete the `net-global-overrides` ConfigMap and replace the existing certificate and key found in the **global.tlsCrt** and **global.tlsKey** with a new, base64-encoded pair.
```

Comment:
I'm not sure where should I replace this existing certificate and key. In the update process? Create a new CM? It's a bit confusing.

Reply:
You should export the new values to env variables `TLS_CERT` and `TLS_KEY` which are used in the `kyma upgrade ...` command below. 

Anyway, I would recommend to move it to the next step, as it has nothing in common with removing the old ConfigMap.